state_synchronizer/src/coordinator.rs

```diff
@@ -143,9 +146,10 @@ impl<T: ExecutorProxyTrait> SyncCoordinator<T> {
                                         }
                                     }
                                     if message.has_chunk_response() {
-                                    if let Err(err) = self.process_chunk_response(message.take_chunk_response()).await {
+                                    if let Err(err) = self.process_chunk_response(&peer_id, message.take_chunk_response()).await {
                                             error!("[state sync] failed to process chunk response: {:?}", err);
                                         }
+                                        self.peer_manager.update_score(&peer_id, PeerScoreUpdateType::Success);
```

Comment:
why do we update score as success unconditionally ?

Reply:
oops was supposed to be in the else